zerver/tests/test_signup.py

```diff
@@ -2379,6 +2380,36 @@ def test_signup_with_full_name(self) -> None:
             ["Enter your account details to complete registration."], result
         )
 
+    def test_signup_with_email_address_race(self) -> None:
+        """
+        Check if signing up without a full name redirects to a registration
+        form.
+        """
```

Comment:
I think the docstring is left over from another test. We probably want a brief explanation of the race condition here.

We actually have 3 occurrences of this docstring in this file, and 2 of them look to not make sense :neutral_face: Can you add a quick commit to delete them while we're at this?